capa/capabilities/static.py

```diff
@@ -96,7 +98,7 @@ def find_basic_block_capabilities(
 
 def find_code_capabilities(
     ruleset: RuleSet, extractor: StaticFeatureExtractor, fh: FunctionHandle
-) -> Tuple[MatchResults, MatchResults, MatchResults, int]:
+) -> Tuple[MatchResults, MatchResults, MatchResults, FeatureSet]:
```

Comment:
changing the signature of a function is a breaking change, so this should wait until the next major release.